.gitignore

```diff
@@ -66,3 +66,6 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
+
+# Varios
```

Comment:
Nitpick: typo

Reply:
Oops, that should be `Various`. I'll update that in the next PR.